### PR TITLE
Remove custom retry handler, now handled by pulpcore

### PR DESCRIPTION
**Pull request by dralley**
[noissue]

### CODE REVIEW

**Comment by dralley** [comment]
You probably aren't ready to drop <3.14 support for a while, but that's fine, it can wait.

**Review by fao89 — COMMENTED** [review]
I believe if you rebase the CI will pass this time

**Comment by dralley** [comment]
@fao89 Thanks

**Review by fao89 — APPROVED** [review]
LGTM

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-async_lru
+async_lru~=1.0
```
[the remark this answers is not in the document]
Comment (fao89) [reply requirements.txt:1]: I think you can consider the one from 0.14 branch

**Pull request merged**